core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildRunner.java

```diff
@@ -41,13 +43,15 @@ public void setup(boolean processInheritIODisabled) {
     public void cleanupServer(File outputDir, boolean processInheritIODisabled) throws InterruptedException, IOException {
     }
 
-    public int build(List<String> args, Path outputDir, boolean processInheritIODisabled)
+    public int build(List<String> args, Logger log, Path outputDir, boolean processInheritIODisabled)
```

Comment:
Is there any reason why the log is being passed in and it's not a field of the class itself?
This will lead to the log showing the wrong source class for the messages inside this class

Reply:
> Is there any reason why the log is being passed in and it's not a field of the class itself?

Only my lack of experience when it comes to using Loggers :(